resource-managers/kubernetes/core/pom.xml

```diff
@@ -51,7 +51,16 @@
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
       <type>test-jar</type>
     </dependency>
-
+    <dependency>
```

Comment:
Volcano support in k8s-cli would be released at kubernetes-client v5.11
https://github.com/fabric8io/kubernetes-client/pull/3580